Service/DeveloperOverheidService.php

```diff
@@ -179,7 +213,7 @@ public function importRepository($repository): ?ObjectEntity
 
         isset($this->io) && $this->io->success('Checking repository '.$repository['name']);
         $synchronization = $this->synchronizationService->findSyncBySource($source, $repositoryEntity, $repository['id']);
-        $synchronization = $this->synchronizationService->synchronize($synchronization, $repository);
+        $synchronization = $this->synchronizationService->handleSync($synchronization, $repository);
```

Comment:
Is there a reason to revert to handleSync? The synchronize function should replace handleSync in order to force remapping if the mapping changes.